src/components/Room.css

```diff
@@ -0,0 +1,33 @@
+.room-container {
+    display: inline-block;
+    padding-left: 20px;
```

Comment:
If we're going to use `20px`, `30px`, `40px` in alot of places, can we extract these out with a variable name like `standard-padding`, or whatever the type of padding is?

Reply:
Yea we could do something like this in `App.css` (available to all components):

```
.standard-padding {
    padding-left: 20px;
}
```

and then in a container like `room-container`:

```
<div className="room-container standard-padding"/>
```

Is this what you meant?

Reply:
Oh I forgot we're using regular css.. In scss you can declare variables like `$my-variable: 30px;`, and then use it like
```
.my-class {
  padding: $my-variable;
}
```